pcs/resource.py

```diff
@@ -527,6 +557,12 @@ def resource_list_options(
         agent_name = find_single_agent(
             lib.resource_agent.get_agents_list().names, agent_name_str
         )
+    if agent_name.standard == "stonith":
+        deprecation_warning(
+            "Accepting of stonith/fencing agents is deprecated and will be "
+            "removed in a future version. Please use 'pcs stontith describe' "
+            "instead."
```

Comment:
I would change the wording to be the same as in the reports package:
Ability of this command to accept stonith/fencing agents is deprecated and will be removed in a future release. Please use 'pcs stontith describe' instead.